docs/Windows_Installation_Instructions.md

```diff
@@ -0,0 +1,176 @@
+# Instructions for Windows
+
+This document describes how to configure and start up a running version of MMS 4.
+
+Versions Used to Test these instructions
+* [MMS v4.0.0-a2](https://github.com/Open-MBEE/mms/tree/4.0.0-a2)
+* Windows 10 (64-bit) 10 Enterprise (v 10.0.17763 Build 17763)
+* Docker Desktop (Community Edition) v2.3.0.5
+  * Docker Engine v19.03.12
+  * Docker-Compose v1.27.2
+
+# Steps
+
+## 1. Download and install Docker Desktop
+> If you already have a fairly modern version of Docker Desktop installed ***you can skip this step***.
+
+Download the [Docker Desktop Community 2.3.0.5](https://docs.docker.com/docker-for-windows/release-notes/#docker-desktop-community-2305) installer and follow the installation instructions.
+
+## 2. Download and install Git for Windows
+> If you already have Git ***you can skip this step***.
+
+Go to the [Git for Windows](https://gitforwindows.org/) site and download the latest version.
+
+This will not only install Git, but will also provide Git BASH, a shell command for working with Git in Windows.
+
+## 3. Clone the MMS 4 Repository
+> If you already cloned the repository, make sure you are on the correct branch/tag by following the checkout step (4).
+
+Clone the repository and checkout the latest tagged version, as of the writing of these instructions.
+
+1. Open GitBash
+
+1. Navigate to the folder you want to clone the repo on, e.g., `C:\repos\mms`, but you may need to clone in your user folder if you do not have permission to create a new folder at the root drive level, ***for example***:
+
+    ```bash
+    cd /c/repos/mms
+    ```
+    
+    > If you have to use a deeply nested folder, make sure you check out the section below on how to extend the Windows maximum path length.
+
+1. Clone the `mms` repo
+
+    ```bash
+    git clone git@github.com:Open-MBEE/mms.git
+    ```
+
+1. Checkout the right branch/tag:
+
+    ```bash
+    git checkout 4.0.0-a2
+    ```
+
+## 4. Get setup
+> IMPORTANT: Windows PowerShell is the recommended shell for the remaining steps.
+
+1. Open "Windows PowerShell".
+1. Navigate to the folder where you cloned the `mms` repository.
+1. Copy the example MMS configuration file
+   * `cp .\example\src\main\resources\application-test.properties .\example\src\main\resources\application.properties`
+   > This configuration file is intended to work with the `docker-compose` file in the MMS repo
+1. Make sure that `docker-compose` is available by running:
+
+    ```bash
+    docker-compose --version
+    ```
+    You should see something like this:
+    ```bash
+    docker-compose version 1.27.2, build 18f557f9
+    ```
+
+## 5. Start and Monitor MMS
+1. Start up the containers
+
+    ```bash
+    docker-compose up -d
+    ```
+
+    You should see something like this:
+
+    ```bash
+    Creating network "mms_default" with the default driver
+    Creating mms_postgres_1      ... done
+    Creating mms_elasticsearch_1 ... done
+    Creating mms                 ... done
```

Comment:
Not super important but we may need to add a new line for `minio`, something like: `    Creating mms_minio_1 ... done` (double checking that is what it looks like when cloning from the new `4.0.0-b2` tag.